### ISSUE TITLE
--xtflog: suppress geometry

### ISSUE THREAD

**Issue by beistehen** [claeis/ilivalidator#394]
Gelegentlich wäre es hilfreich, wenn die Geometrie in IliVErrors bewusst nicht exportiert würde. Ein Anwendungsfall ist, wenn man das xtflog im Anschluss an die Validierung in einem GIS mit den Ursprungsdaten über die OID joined und dann sowohl eine Ursprungsgeometrie als auch eine Geometrie aus dem xtflog hat.

**Comment by beistehen** [comment]
Ja, das stimmt. Wenn die Geometrie jedoch von Anfang an nicht dabei ist, so muss man sich darum nicht kümmern.
Wie gesagt, _gelegentlich wäre es hilfreich ..._

**Comment by edigonzales** [comment]
Schreibt man Selects sowieso nicht in der Form "SELECT attr1, attr2 FROM ..." und nicht bloss "SELECT * FROM ..."? ;)
